ocw/data_source/PODAAC.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@lewismc I willingly did this, because if I used podaac.py then there is a clash during importing podaac from podaacpy, since both the files have same name. Let's say I have changed the file name to podaac.py in ocw and import podaac inside it, it is importing itself rather than importing podaac from podaacpy. That was strange.